{"url": "https://towardsdatascience.com/the-subsets-powerset-of-a-set-in-python-3-18e06bd85678", "time": 1683019441.79746, "path": "towardsdatascience.com/the-subsets-powerset-of-a-set-in-python-3-18e06bd85678/", "scrape_status": {"code": 4, "message": "invalid response status code", "resp_status_code": 429}}